lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    dose_difference = dose_evaluation - dose_reference
</pre><p>There might be some clever optimisations here, but probably unneeded.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>